connectors/slack/template/storage.js

```diff
@@ -1,47 +1,69 @@
 const Sdk = require('@fusebit/add-on-sdk');
 
-let storage;
-module.exports.ensureStorage = (ctx) => {
-  const twoMinutesFromNow = Date.now() + 2 * 60 * 1000;
-  if (!storage || storage.expiry < twoMinutesFromNow) {
-    storage = Sdk.getStorageClient(ctx);
-  }
-};
-
 let mappingTimestamp;
-module.exports.get = async () => {
+let etag;
+
+async function getStorageClient(ctx) {
+  return await Sdk.createStorageClient(
+    ctx,
+    ctx.fusebit.functionAccessToken,
+    `boundary/${ctx.boundaryId}/function/${ctx.functionId}`
```

Comment:
See my previous comment. This should likely be scoped to `boundary/${ctx.boundaryId}/function/${ctx.functionId}/root` or some such root document.

Reply:
Stays unchanged